FCCee/FullSim/ALLEGRO/ALLEGRO_o1_v03/run_digi_reco.py

```diff
@@ -98,6 +98,7 @@
 
 # run photon ID algorithm
 runPhotonIDTool = False
+logEWeightInPhotonID = False and runPhotonIDTool
```

Comment:
Hi @dasphy ,
I don't think that the flag logEWeightInPhotonID should depend on runPhotonIDTool.
The runPhotonIDTool flag is used to decide whether we want to run the photon ID BDT and save its output score in the output ROOT file, while logEWeightInPhotonID is used to decide if we want to use logE or E weights when calculating some shape variables used to decorate the clusters, no matter whether we run or not the photon ID tool later. I would just set the flag to False